tests/e2e_test.py

```diff
@@ -18,13 +19,7 @@
 from tests.test_utils import QasmFile, get_qasm_as_circuit
 
 if __name__ == "__main__":
-    machine = Machine(
-        3,
-        {1},
-        3.0,
-        1.0,
-        2.0,
-    )
+    machine = Machine(3, {OpType.TK1, OpType.TK2}, {1}, MachineTimings(3.0, 1.0, 2.0))
```

Comment:
Does it make sense to add `tk_gateset` as a constant under the `Machine` module?

This test is somewhat of dead code as it's currently not exercised with `make tests`, so I don't really care as much. However, I was thinking of adding `python tests/e2e_test.py` under the `make tests` command.

Reply:
I changed this to use `QTM_DEFAULT_GATESET` instead.